internal/driver/config.go

```diff
@@ -157,6 +158,7 @@ func init() {
 		"sort":                 "sort",
 		"granularity":          "g",
 		"noinlines":            "noinlines",
+		"columns":              "columns",
```

Comment:
Maybe name it "showcolumns" so that `-lines -showcolumns` looks a bit better?

Reply:
Done.